plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -99,10 +99,11 @@ func createHandlerWithConfig(kubeClient kubernetes.Interface, informerFactory in
 	}
 	quotaConfiguration := install.NewQuotaConfigurationForAdmission()
 
-	handler, err := resourcequota.NewResourceQuota(config, 5, stopCh)
+	handler, err := resourcequota.NewResourceQuota(config, 5)
 	if err != nil {
 		return nil, err
 	}
+	handler.SetDrainedNotification(stopCh)
 	handler.SetExternalKubeClientSet(kubeClient)
 	handler.SetExternalKubeInformerFactory(informerFactory)
 	handler.SetQuotaConfiguration(quotaConfiguration)
```

Comment:
can you switch these to using the initializer/validate pattern to make sure our tests match reality in terms in invocation order and that they are valid?

```go
	pluginInitializer := genericadmissioninitializer.New(...)
	pluginInitializer.Initialize(handler)
	err := admission.ValidateInitialization(handler)
  ...
```

Reply:
We unfortunately need more than one initializer (there are two in place).
That said - I switched to using that pattern here and in integration test.